goal_src/jak2/dgos/std.gd

```diff
@@ -0,0 +1,4 @@
+("STD.DGO"
+  ("race-h.o" "race-h")
+  ("race-info.o" "race-info")
```

Comment:
Is this actually the full list?

Reply:
No, forgot about this -- I'll fix this properly